resources/views/layouts/skeleton.blade.php

```diff
@@ -23,6 +23,7 @@
           'profileDefaultView' => auth()->user()->profile_active_tab,
       ]); !!}
     </script>
+    <base href="{{ url('/') }}/" />
```

Comment:
Should be move to top of head otherwise the favicon url is wrong.

Reply:
Done